api/cluster-resources/build.rb

```diff
@@ -27,6 +27,9 @@ def build_snippets_menu()
 
   FileUtils.mkdir_p "generated"
   File.write("generated/aou-snippets-menu.js", tmpl)
```

Comment:
I don't think this one will be needed anymore.

Reply:
The ticket states:
```
Note: cluster-resources must continue to be pushed to GCS following this change. Teardown of this approach will follow later.
```

I wanted to do minimal changes to that deploy for now, and refactor when we removed that as a requirement. I think we still need this if we want to push that file to GCS.